apple/Sources/FerrostarSwiftUI/Views/SpeedLimit/SpeedLimitView.swift

```diff
@@ -0,0 +1,47 @@
+import SwiftUI
+
+public struct SpeedLimitView: View {
+    @Environment(\.locale) private var locale
+
+    var speedLimit: Measurement<UnitSpeed>
+    var valueFormatter: NumberFormatter
+    var unitFormatter: MeasurementFormatter
+
+    public init(
+        speedLimit: Measurement<UnitSpeed>,
+        valueFormatter: NumberFormatter = DefaultFormatters.speedFormatter,
+        unitFormatter: MeasurementFormatter = DefaultFormatters.speedWithUnitsFormatter
+    ) {
+        self.speedLimit = speedLimit
+        self.valueFormatter = valueFormatter
+        self.unitFormatter = unitFormatter
+    }
+
+    public var body: some View {
+        switch locale.identifier {
+        case "en_US":
+            USSpeedLimitView(
+                speedLimit: speedLimit,
+                valueFormatter: valueFormatter,
+                unitFormatter: unitFormatter
+            )
+        default:
+            ROWSpeedLimitView(
+                speedLimit: speedLimit,
+                valueFormatter: valueFormatter,
+                unitFormatter: unitFormatter
+            )
+        }
```

Comment:
I am not sure this is the best way to go about deciding which speed limit to show. For one, [the US isn't the only country to use mph for speed limits](https://www.rhinocarhire.com/Drive-Smart-Blog/Which-Countries-use-MPH-or-KPH.aspx).

Second, I'm pretty convinced that locale is not the correct way to set speed display preference.

I think we can punt on the specifics of this for the moment and:

1. Have it be a pair of preferences for the moment (maybe bundled into the theme / display units / formatter stuff??). It's really a pair of preferences: do you want the US/MUTCD style or Vienna Convention style that's used *almost* everywhere else. If I were designing this, I'd make it vary on geography, but that's way out of scope for now ;)
2. Move the logic at the use site (get rid of this view wrapper until we have a better idea of how users want to determine it automatically).